apps/deploy-web/src/components/onboarding/steps/EmailVerificationStep/EmailVerificationStep.spec.tsx

```
@@ -0,0 +1,151 @@
+  it("displays verify error alert", () => {
+    setup({ verifyError: "Invalid verification code" });
+
+    expect(screen.queryByText("Invalid verification code")).toBeInTheDocument();
+  });
+
+  it("displays verifying state", () => {
+    setup({ isVerifying: true });
+
+    expect(screen.queryByText("Verifying...")).toBeInTheDocument();
+    const inputs = screen.getAllByRole("textbox");
+    inputs.forEach(input => expect(input).toBeDisabled());
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)